consent-ws/src/main/resources/assets/api-docs.yaml

```diff
@@ -2049,6 +2049,55 @@ paths:
           description: The requested user doesnt exist in the system
           schema:
             $ref: '#/definitions/ErrorResponse'
+  /api/nih-login/{userId}:
+    delete:
+      summary: Delete Era Commons Account
+      description: Removes only Era Commons Account related information from the given user in its Researcher Profile
+      parameters:
+        - name: userId
+          in: path
+          required: true
+          type: integer
+          format: int32
+      tags:
+        - User
+        - Researcher
+        - Era Commons
```

Comment:
nit: Technically, it's `eRA Commons`. See https://public.era.nih.gov/commons/public/login.do and  https://era.nih.gov/